src/app/vm/shared/vm.model.ts

```diff
@@ -63,6 +63,8 @@ export const isHttpAuthMode = (vm: VirtualMachine) => {
   return mode && vm.state === VmState.Running;
 };
 
+export const ColorDelimeter = ';';
```

Comment:
Should this line be in this file?

Reply:
why not? this value was here before

Reply:
How it related to VM model? It uses only in vm-tag.service.ts, it probably better to put it there?